lib/routes/routesUtils.js

```diff
@@ -100,11 +100,19 @@ function errorXMLResponse(errCode, response, log, corsHeaders) {
     log.addDefaultFields({
         bytesSent,
     });
+    if (corsHeaders) {
+        // eslint-disable-next-line no-param-reassign
+        corsHeaders['Content-Type'] = 'application/xml';
+        // eslint-disable-next-line no-param-reassign
+        corsHeaders['Content-Length'] = xmlStr.length;
+    }
     setCommonResponseHeaders(corsHeaders, response, log);
     response.writeHead(errCode.code, { 'Content-type': 'application/xml' });
     return response.end(xmlStr, 'utf8', () => {
         log.end().info('responded with error XML', {
             httpCode: response.statusCode,
+            xmlStr,
+            corsHeaders,
```

Comment:
please remove these two lines.  I did a subsequent pr on master to remove these.